docs/src/tutorial/example-data.rst

```diff
@@ -51,7 +51,7 @@ The workflow can take several minutes to run. While it is running, you can learn
 - ``nextstrain build .``
    - This tells the :term:`docs.nextstrain.org:Nextstrain CLI` to :term:`build <docs.nextstrain.org:build (verb)>` the workflow from ``.``, the current directory. All subsequent command-line parameters are passed to the workflow manager, Snakemake.
 - ``--cores all``
-   - This required Snakemake parameter specifies the number of CPU cores to use (:doc:`more info <snakemake:executing/cli>`).
+   - This optional parameter (not shown) can be passed to Snakemake specifying the number of CPU cores to use (:doc:`more info <snakemake:executing/cli>`). Nextstrain will pass a `--cores all` by default.
```

Comment:
I'd be in favor of removing this entirely to keep the tutorial simple. It's an additional detail that users can look up if they want, among other optional Snakemake parameters.

Reply:
Thanks! Normally I'd do a "Commit suggestion" but hit an "Applying suggestion on deleted lines is not currently supported. 

![Screen Shot 2022-08-25 at 3 26 54 PM](https://user-images.githubusercontent.com/1077254/186780087-8fd42cb4-480d-445c-9005-0dc1dfc6e8d7.png)

So I added the changes in a separate commit.

Reply:
I went around the disabled suggestions button by manually adding an empty suggestions block:

    ```suggestions
    ```

It rendered fine which was the main intent.

I think it'd be good to use a more descriptive title for 6b4201d1a3eda612f22300acf60cffa1f28c539b or even better, squash it into https://github.com/nextstrain/ncov/pull/997/commits/3095b26da23cd020335658857ca44fceb00dc77e.